packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1250,22 +1250,35 @@ abstract class ResidentRunner {
     }
   }
 
-  Future<bool> launchDevTools({bool openInBrowser = false}) async {
+  DevToolsServerAddress activeDevToolsServer() {
+    _devToolsLauncher ??= DevtoolsLauncher.instance;
+    return _devToolsLauncher.activeDevToolsServer;
+  }
+
+  Future<void> serveDevToolsGracefully({
+    Uri devToolsServerAddress,
+    bool openInBrowser = false,
```

Comment:
What is the functionality of `openInBrowser` now? Can we let the user click on the link themselves so they get their preferred browser/existing instance

Reply:
Good question. I'm following up with @helin24 on this to see what we do from IntelliJ.

Reply:
Added a TODO to remove this in a cleanup CL, as it will require a g3fix, and I'd like to get this landed sooner than later to start working on iterative CLs. AFAIK the 'v' hotkey was the only place using this functionality `openInBrowser: true`, so removing the param should be safe

Reply:
Sounds reasonable!